lib/Analyze.cpp

```diff
@@ -905,13 +936,41 @@ void RecoverMemoryAccesses(
     // TODO(pag): Leaving these as integers might be best, as we may go an
     //            collect them in the optimizer.
     } else {
-      LOG(ERROR) << "TODO: Found byte address " << std::hex << addr << std::dec
-                 << " that is mapped to memory but doesn't directly "
-                 << "resolve to a function or variable";
-
+      if (addr) {
+        LOG(ERROR) << "TODO: Found byte address " << std::hex << addr
+                   << std::dec
+                   << " that is mapped to memory but doesn't directly "
+                   << "resolve to a function or variable";
+        new_val = llvm::ConstantInt::get(int_type, addr, false);
+      } else {
+        new_val = used_val;
+      }
       new_val = llvm::ConstantInt::get(int_type, addr, false);
-    }
+      if (type) {
```

Comment:
A comment here that gives an example of what bitcode gets produced by this particular transform, and why.